EduID/Flows/CreateEduID/ViewControllers/CheckEmailViewController.swift

```diff
@@ -80,8 +82,17 @@ class CheckEmailViewController: CreateEduIDBaseViewController {
         
         // - add constraints
         stack.edgesToSuperview(insets: TinyEdgeInsets(top: 24, left: 24, bottom: 24, right: 24), usingSafeArea: true)
-        
-        posterLabel.height(34)
+        gmailButton.addTarget(self, action: #selector(openWeb(for:)), for: .touchUpInside)
+        outlookButton.addTarget(self, action: #selector(openWeb(for:)), for: .touchUpInside)
 
     }
+    
+    @objc private func openWeb(for sender: UIButton) {
+        switch sender.tag {
+        case 0: UIApplication.shared.open(URL(string: "https://gmail.com")!)
+        case 1: UIApplication.shared.open(URL(string: "https://login.live.com")!)
```

Comment:
This is not the url for outlook web access

Reply:
sure, do you have the correct URL? of course I can look it up myself, but just to be on the safe side.

Reply:
I assume `https://outlook.office.com` is the one?